src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsDetailsMapping.java

```diff
@@ -172,11 +117,6 @@ public static List<String> getElementsDisputed(SscsCaseData caseData) {
             .collect(Collectors.toList());
     }
```

Comment:
Can you also move getElementsDisputed to HearingsDurationMapping as it is only for getHearingDuration